### PR TITLE
Morph uses appearance

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**

### CODE REVIEW

File: code/game/gamemodes/miniantags/morph/morph.dm [hunk 1]
```diff
@@ -76,22 +76,10 @@
...
-	//anim(loc,src,'icons/mob/mob.dmi',,"morph",,src.dir) No effect better than shit effect
-
-	//Todo : update to .appearance once 508 hits
-	if(ishuman(target))
-		var/mob/living/carbon/human/H = target
-		overlays = H.get_overlays_copy(list(L_HAND_LAYER,R_HAND_LAYER))
-	else
-		overlays = target.overlays.Copy()
-
-	visible_message("<span class='warning'>[src] suddenly twists and changes shape, becoming a copy of [target]!</span>", \
```
Comment (RemieRichards) [inline code/game/gamemodes/miniantags/morph/morph.dm]: why'd you remove the message nerd?

Comment (KorPhaeron) [inline code/game/gamemodes/miniantags/morph/morph.dm]: Good question

**Comment by KorPhaeron** [comment]
Teach me to edit on my phone I'll fix it later

File: code/game/gamemodes/miniantags/morph/morph.dm [hunk 2]
```diff
@@ -76,24 +76,12 @@
...
-	if(ishuman(target))
-		var/mob/living/carbon/human/H = target
-		overlays = H.get_overlays_copy(list(L_HAND_LAYER,R_HAND_LAYER))
-	else
-		overlays = target.overlays.Copy()
-
+	appearance = target.appearance
+	transform = matrix()
+	pixel_y = 0
+	pixel_x = 0
```
Comment (tkdrg) [inline code/game/gamemodes/miniantags/morph/morph.dm]: why not initial(transform), initial(pixel_y), etc.

Comment (KorPhaeron) [inline code/game/gamemodes/miniantags/morph/morph.dm]: The morph's initial is always 0 for pixel_x and pixel_y, didn't think it was needed.

**Pull request merged**